csvdiff/nix/packages.nix

```diff
@@ -0,0 +1,37 @@
+# SPDX-FileCopyrightText: 2023 Technology Innovation Institute (TII)
+#
+# SPDX-License-Identifier: Apache-2.0
+_: {
+  perSystem = {
+    pkgs,
+    lib,
+    ...
+  }: let
+    pp = pkgs.python3Packages;
+  in {
+    packages = rec {
+      default = csvdiff;
+
+      csvdiff = pp.buildPythonPackage rec {
```

Comment:
same

Reply:
True, this inner rec isn't needed. Fixed with: https://github.com/tiiuae/ci-public/pull/212/commits/24296a2bd5d7d96bdcfe6c68c2a162ec45f1cc6c